cmd/flags/prover.go

```diff
@@ -134,6 +134,12 @@ var (
 		Value:    uint64(time.Hour.Seconds()),
 		Category: proverCategory,
 	}
+	TempCapacityExpiresAt = &cli.DurationFlag{
+		Name:     "prover.tempCapacityExpiresAt",
+		Usage:    "time in seconds temporary capacity lives for, format: 12s",
+		Value:    12 * time.Second,
```

Comment:
Shall we make this default value larger? like 36 seconds. Thinking 3 block time will be a safer time to ensure a tx to be included?